### PR TITLE
Resolved the merge conflict

### PR DESCRIPTION
**Pull request by kishor3107**
@mantovani @badrinathpatchikolla  please review.

### CODE REVIEW

**Review by mantovani — CHANGES_REQUESTED** [review]
Let's also upgrade almaren for the latest version.

**Comment by kishor3107** [comment]
@mantovani  Updated.

File: build.sbt [hunk 1]
```diff
@@ -14,8 +14,8 @@ scalacOptions ++= Seq("-deprecation", "-feature")
 libraryDependencies ++= Seq(
   "org.apache.spark" %% "spark-core" % sparkVersion % "provided",
   "org.apache.spark" %% "spark-sql" % sparkVersion % "provided",
-  "com.github.music-of-the-ainur" %% "almaren-framework" % "0.2.8-2.4.0" % "provided",
-  "com.google.cloud.spark" %% "spark-bigquery-with-dependencies" % "0.18.1",
+  "com.github.music-of-the-ainur" %% "almaren-framework" % "2.4.5-2.4.5" % "provided",
```
Comment (badrinathpatchikolla) [inline build.sbt]: This is not the latest version of Almaren

**Review by badrinathpatchikolla — APPROVED** [review]

**Review by badrinathpatchikolla — APPROVED** [review]

**Comment by badrinathpatchikolla** [comment]
@mantovani  Please review and release 0.0.3 version for BigQuery Almaren

**Review by mantovani — APPROVED** [review]

**Pull request merged**